common/src/test/java/tachyon/util/io/BufferUtilsTest.java

```diff
@@ -383,10 +383,15 @@ public void sliceByteBufferTest() {
       // Slice a ByteBuffer from the target position to the end
       int slicedBufferLength = size - slicePosition;
       slicedBuffer = BufferUtils.sliceByteBuffer(buf, slicePosition, slicedBufferLength);
+      ByteBuffer slicedBuffer1 = BufferUtils.sliceByteBuffer(buf, slicePosition);
```

Comment:
also since we have the overloading method, the javadoc of this test should be updated, too